ui-admin/src/study/surveys/PreEnrollView.tsx

```diff
@@ -28,18 +29,17 @@ function RawPreEnrollView({ studyEnvContext, survey, readOnly = false }:
   /** saves as a new version and updates the study environment accordingly */
   async function createNewVersion({ content: updatedContent }: { content: string }): Promise<void> {
     survey.content = updatedContent
-    try {
+    doApiLoad(async () => {
```

Comment:
Just based on this diff, it looks like the amount of code here is exactly the same, just replacing `try`/`catch` with `doApiLoad`. Are we gaining any new behavior from `doApiLoad` that makes this change more of a win?

Reply:
here, no, just better standardization